doc/usage.rst

```diff
@@ -138,6 +138,25 @@ The *key-string* is a string of 16 characters which is used to
 encrypt each file of Python byte-code before it is stored in
 the archive inside the executable file.
 
+.. _extracting libraries:
+
+Extracting Libraries
```

Comment:
Can you think about a better headline. This is not only about "libraries". Please use the wording we use when describing _MEIxxxx and make clear that `_MEIxxxx` will created within die directory passed here.